src/Pages/wonkeunchoi/Main/Main.scss

```
@@ -0,0 +1,175 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.Main_wonkeunchoi {
+  width: 100%;
+  background-color: #fafafa;
+
+  section {
+    display: flex;
+    margin: 0 auto;
+    top: 100px;
+    max-width: 990px;
+  }
+
+  section .left_layout {
+    width: 100%;
+    height: 1000px;
+    top: 100px;
+    border: 1px solid darkgray;
+    border-radius: 5px;
+    background-color: white;
+  }
+
+  section .right_layout {
+    position: sticky;
+    margin-left: 3%;
+    width: 500px;
+    top: 100px;
+    height: 100%;
+    background-color: #fafafa;
+  }
+
+  .top_emtied_box {
+    display: block;
+    width: 100%;
+    height: 100px;
+  }
+
+  .feed_title {
+    position: relative;
+    width: 100%;
+    height: 56px;
+    background-color: white;
+    border: 1px solid rgb(241, 238, 238);
+    border-radius: 3px;
+  }
+
+  .profile_small_img {
+    width: 35px;
+    height: 35px;
+    margin-left: 15px;
+    margin-top: 10px;
+    border: 1px solid darkgray;
+    border-radius: 18px;
+  }
+
+  .feed_image {
+    width: 100%;
+    height: 100%;
+  }
+
+  .profile_box {
+    position: absolute;
+    width: 636px;
+    height: 650px;
+  }
+
+  .nickname {
+    position: absolute;
+    display: inline;
+    font-weight: bold;
+    width: 100%;
+    margin-top: 3.5%;
+    margin-left: 2%;
+  }
+
+  .nickname #dot_image {
+    position: absolute;
+    width: 21px;
+    height: 16px;
+    margin-left: 67%;
+  }
+
+  .post_box {
+    display: inline;
+    border: 1px solid darkgrey;
+    width: 100%;
+    height: auto;
+  }
+
+  .post {
+    width: 90%;
+    height: 8%;
+    margin: 19px;
+  }
+
+  .comment_storage {
+    width: 90%;
+  }
+
+  /* /----------------------------------- */
+
+  .right_layout #side_profile {
+    width: 55px;
+    height: 55px;
+    border-radius: 30px;
+    border: 0.5px solid darkgrey;
+  }
+
+  .right_layout .follow_myInfo {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+  }
+
+  .right_layout .follow_myInfo p {
+    display: block;
+    margin: 5px;
+    font-size: 14px;
+    font-weight: bold;
+  }
+
+  #myName_box {
+    width: 65%;
+    margin-top: 8px;
+    margin-left: 12px;
+  }
+
+  #myName {
+    font-weight: inherit;
+    color: #8e8e8e;
+  }
+
+  .follow_myInfo span {
+    font-size: 14px;
+    font-weight: bold;
+    color: #0095f6;
+  }
+
+  .follow_box {
+    width: 96%;
+    height: 350px;
+    margin-top: 30px;
+  }
+
+  .right_layout .follow_box #user_recommend_font {
+    margin-right: 40%;
+    font-size: 17px;
+    color: #8e8e8e;
+    font-weight: inherit;
+  }
```

```suggestion
  .right_layout{ 
  
    .follow_box {
    
      #user_recommend_font {
        margin-right: 40%;
        font-size: 17px;
        color: #8e8e8e;
        font-weight: inherit;
      }
    }
}
```
지금은 클래스와 아이디 이름을 다 써줬지만, Sass의 depth를 이용한다면 굳이 길게 클래스명과 아이디를 쓰지 않아도 표현할 수 있을 것 같습니다!